design/commands.md

```diff
@@ -103,9 +103,9 @@ $ sd_cmd exec foo/bar@1 -baz sample
  - **binary** - A script or binary that can be downloaded and directly executed.
 
 ```bash
-$ curl -o /opt/sd/commands/foo/bar/1.0.1 https://store/v1/commands/foo/bar/1.0.1
-$ chmod +x /opt/sd/commands/foo/bar/1.0.1
-$ /opt/sd/commands/foo/bar/1.0.1 -baz sample
+$ curl -o /opt/sd/commands/foo/bar/1.0.1 https://store/v1/commands/foo/bar/1.0.1/foobar.sh
```

Comment:
The path for store API wouldn't be changed. On the other hand, you need to modify the argument for `-o` switch.

Reply:
You're right I will fix.